src/components/featured-article/_test-examples.js

```diff
@@ -0,0 +1,70 @@
+export const EXAMPLE_FEATURED_ARTICLE_WITH_CHART = {
+    title: {
+        text: 'Economic Trends 2024',
+        url: '/economy/economic-trends-2024',
+    },
+    metadata: {
+        date: {
+            prefix: 'Released',
+            showPrefix: true,
+            iso: '2024-05-01',
+            short: '1 May 2024',
+        },
+        object: {
+            text: 'Bulletin',
+        },
```

Comment:
If object only has the text param should we flatten this rather than have an object? Is there the possibility that there is more than one object? If there is we could just use an array instead.